D:\a\_work\1\s\bin\dotnet\packs\Microsoft.Maui.Sdk\6.0.542-ci.net6.0.17791\Sdk\AutoImport.props(5,14): error MSB4184: The expression "[MSBuild]::VersionEquals('', 6.0)" cannot be evaluated. Version string was not in a correct format. [D:\a\_work\1\s\src\Controls\samples\Controls.Sample\Maui.Controls.Sample.csproj]

--- a/src/Workload/Microsoft.Maui.Sdk/Sdk/AutoImport.in.props
+++ b/src/Workload/Microsoft.Maui.Sdk/Sdk/AutoImport.in.props
@@ -2,7 +2,7 @@
 <!-- all <ItemGroup/>'s must include $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@'))  -->
 <Project>
 
-  <ItemGroup Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true') and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@')) ">
+  <ItemGroup Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true') and '$(TargetFrameworkVersion)' != '' and $([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@')) ">
     <EolWorkload Include="maui" Url="https://aka.ms/maui-support-policy" />
   </ItemGroup>
 

